js/Source/Delegators/Delegator.SubmitOnChange.js

```diff
@@ -15,16 +15,18 @@ provides: [Delegator.SubmitOnChange]
 
 Delegator.register('change', 'submitOnChange', {
   defaults: {
-    onlyOnce: true
+    onlyOnce: true,
+    spin: false
   },
   handler: function(event, element, api){
     var form = element;
     if (api.get('target')) form = api.getElement('target');
     if (api.get('onlyIfSet') && !element.get('value')) return;
     if (!api.getAs(Boolean, 'onlyOnce') || (api.get('onlyOnce') && !form.retrieve('submitted'))){
       form.fireEvent('submit').submit();
+      if (api.getAs(Boolean, 'spin')) form.spin();
```

Comment:
Sorry for the late response; didn't see this one.

Note that there's already the delegator spinOnSubmit...

Regardless, please update the docs table in the html example to include this option.